tests/superphot_plus/diff_test.py

```diff
@@ -149,7 +149,7 @@ def compare_two_files(file_name, goldens_dir, temp_results_dir):
         True if files are sufficiently similar; False otherwise."""
 
     # Set up
-    deltas = {"dynesty": 0.1, "svi": 0.5, "NUTS": 0.1}
+    deltas = {"dynesty": 0.1, "svi": 0.1, "NUTS": 0.1}
```

Comment:
```suggestion
    deltas = {"dynesty": 0.25, "svi": 0.1, "NUTS": 0.1}
```
The unit test that failed just now was dynesty getting a 0.11 difference (and iirc sandro ran into a similar case of dynesty's diff being a little too off, just before the team meeting today). 

I may have been over-optimistic in setting a relative tolerance at 0.1 - maybe it would be good to increase this, at least while we're fine tuning things?

Reply:
Updated.